ui/shared/components/panel/search/constants.js

```diff
@@ -476,12 +485,22 @@ export const SNP_QUALITY_FILTER_FIELDS = [
   },
 ]
 
+export const MITO_QUALITY_FILTER_FIELDS = [
+  {
```

Comment:
can we use a `DividedFormField` component for this similar to how we do SVs so it has the vertical divider?

Reply:
Added.